### PR TITLE
Modernizes & Updates Wabbajack polymorph types/species

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Modernizes polymorph types by adding mob types that never entered the wabbajack list.

Of note:
- An entire new category for clown monsters!
- Added 'bot' types, hivebots, and a small chance for net guardians (this one may be going a bit too far, @ArcaneMusic )
- Added Xenomorph maids
- Added tons of 'animal' types:
- Ants, axolotls, butterbears, snowbears, russianbears, spores, megacarps, moonicorns, evil krebs, flesh spiders, frogs, deer, eyeballs, gondola, lesser gorillas, lightgeist, migos, hatsune migos, all mining mobs, Mooks ( @Ben10Omintrix ), mothroaches, bullterriers, exotic corgies, nars'ians, penguins, ponies, syndicate ponies, rabbits, regal rats, seedlings, mean seedlings, sheep, snake, banded snakes, banded snakes, the 'interesting' types of spiders, duct spiders, Frosty, mannequin?s, ranged stickmen, and living limbs, because they're funny.

Updated a few species' changesource flags:
- Lustrous can be picked in the Pride mirror.
- Silver Scales can be picked in the Pride mirror and wabbajacked into.
- Skeletons can be picked in the Pride mirror. (couldn't they already?)
- Vampires can be picked in the Pride mirror.
- High-Functioning Zombies can be picked in the Magic mirror.
## Why It's Good For The Game

> Modernizes polymorph types by adding mob types that never entered the wabbajack list.

This list has kind of been left to rot for a while, and I wanted to make the chaos polymorph actually chaotic once more by introducing a lot of funny new critters that we've added or neglected to add before.

I have my qualms with the wabbajack proc still, with how it selects one out of all categories and then something inside. That for example means 1/5th of the time you'll get a fucking slime, which is stupid and annoying. But that's for another PR, this just adds mobs to lists.

> Updated a few species' changesource flags

These species either had no real reason not to have these flags or the reasons they had for them were outdated. I may be wrong here, but I don't see a good reason for Funny Ethereals & Funny Lizards to be locked out of the Funny mirror, they aren't that strong. Vampires aren't that good in their current iteration, either, so they go in too.

Skellies & Vamps are still excempt from the magic mirror because they already have spells for that stuff, Splattercasting & Lich respectively

Same reasoning applies for zombies.
## Changelog
:cl:
add: Modernizes polymorph types by adding mob types that never entered the wabbajack list.
balance: Updated a few species' changesource flags
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
> You currently have a negative Fix/Feature pull request delta of -3. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

aw nuts last i checked it was positive............... still this isnt a 'real' feature pr so dont be an anus, fixes coming soon

**Comment by Time-Green** [comment]
Is this… a negative gbp balance/feature PR that *sniff sniff* changes species availability without prior maintainter approval??? How scrumptious

**Comment by carlarctg** [comment]
noooooo

**Comment by ArcaneMusic** [comment]
I didn't have a hand in netguardians just so you know

**Comment by scrunklershrungle932** [comment]
> Lustrous can be picked in the Pride mirror.

> Silver Scales can be picked in the Pride mirror and wabbajacked into.

what're those?

**Comment by Rhials** [comment]
> > Lustrous can be picked in the Pride mirror.
> 
> > Silver Scales can be picked in the Pride mirror and wabbajacked into.
> 
> what're those?

Lustrous are the pirate subspecies of Ethereals. They come with the Bluespace Prophecy trauma which lets them see/use portals that randomly appear near them.

Silverscales are the pirate subspecies of Lizards. They can turn into a statue at will and do not need to breathe.

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -1487,10 +1488,25 @@
...
 				/mob/living/silicon/robot = 200,
 				/mob/living/basic/drone/polymorphed = 200,
+				/mob/living/basic/bot/dedbot = 50,
+				/mob/living/basic/bot/cleanbot = 50,
+				/mob/living/basic/bot/firebot = 50,
+				/mob/living/basic/bot/honkbot = 50,
+				/mob/living/basic/bot/hygienebot = 50,
+				/mob/living/basic/bot/medbot/mysterious = 50,
+				/mob/living/basic/bot/medbot = 50,
+				/mob/living/basic/bot/vibebot = 50,
```
Comment (Ghommie) [inline code/modules/mob/living/living.dm:1506]:
```suggestion
				/mob/living/basic/bot/dedbot = 25,
				/mob/living/basic/bot/cleanbot = 25,
				/mob/living/basic/bot/firebot = 25,
				/mob/living/basic/bot/honkbot = 25,
				/mob/living/basic/bot/hygienebot = 25,
				/mob/living/basic/bot/medbot/mysterious = 12,
				/mob/living/basic/bot/medbot = 13,
				/mob/living/basic/bot/vibebot = 25,
```
I think the sum of the weights of these bots shouldn't be greater than the top two picks.

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Comment by Ghommie** [comment]
Also removed the CL for the grammar, because it's a comment, it isn't even included in the compiled code.

**Review by Ghommie — APPROVED** [review]

**Pull request merged**